src/crypto/CHIPCryptoPAL.h

```diff
@@ -170,6 +170,12 @@ enum class SupportedECPKeyTypes : uint8_t
     ECP256R1 = 0,
 };
 
+enum class SupportedECPKeyTargets : uint8_t
```

Comment:
```suggestion
enum class ECPKeyTarget : uint8_t
```
We typically don't use plural form for types and `Supported` is kinda implicit based on the fact that enum has a limited number of values, so we could use a shorter form.

Reply:
Makes sense, I was just inspired by `SupportedECPKeyTypes` above.